stoiip_prob/main.py

```diff
@@ -1,6 +1,26 @@
+import numpy as np
+# from stoiip_prob import stoiip as stp
```

Comment:
Unnecessary commented code. If you are not using it, delete it. It will be kept in source control if you want to see it afterwards.

Reply:
Done